steve/util.py

```diff
@@ -73,62 +200,31 @@ def _with_config(*args, **kw):
 
 def get_project_config():
     """Finds and opens the config file in the current directory
+      or parent directory
```

Comment:
Nit: This should be dedented two spaces or alternatively reduce the length of the sentence and have it all on one line.